extension/source/components/Button.tsx

```diff
@@ -1,52 +1,26 @@
+import { IconProps } from 'phosphor-react';
 import * as React from 'react';
 
 const Button = (props: {
+  className?: string;
   onPress: () => void;
-  highlight?: boolean;
+  highlight?: boolean; //TODO - remove dependency
```

Comment:
What dependency needs to be removed for this? Would be nice to document in this comment.

Reply:
https://github.com/jzaki/bls-wallet/issues/110

Reply:
added issue description 👍🏼